optaweb-employee-rostering-backend/pom.xml

```diff
@@ -30,7 +30,7 @@
   <description>Spring Boot Backend for OptaWeb Employee Rostering as a Service</description>
 
   <properties>
-    <java.version>1.8</java.version>
+    <java.version>1.11</java.version>
```

Comment:
I believe that this is just `11`.

Reply:
It is just 11, but it still built; odd

Reply:
Perhaps it's not being used anywhere, then?

Reply:
apparently `java.version` is only if you use the spring boot starter parent (we don't). So we should use `<maven.compiler.release>11</maven.compiler.release>` instead